### PR TITLE
Add some translations to cache basics

### PR DESCRIPTION
**Pull request by hnakamur**
This pull request add some translations to the cache basic section.

Please merge this pull request after https://github.com/trafficserver-doc-ja/trafficserver/pull/121.

I read https://github.com/trafficserver-doc-ja/trafficserver-doc-ja#transifex and I am thinking to add/modify translations at transifex after this pull request.
Transifex is the preferred way to add translations now, am I right?

### CODE REVIEW

**Comment by masaori335** [comment]
@hnakamur Dose `gopo` have any support for commented out msgid and msgstr?
IMO, some translations in this PR are already translated. I assume that msgid of those are modified and  those are commented out in old file, then those are not imported in new file.

**Comment by hnakamur** [comment]
@masaori335 Do you mean "Obsolete Messages" (lines starting with "#~")?
* [2.5.3. Obsolete Messages](http://pology.nedohodnik.net/doc/user/en_US/ch-poformat.html)
* [GNU gettext utilities: Obsolete Entries](https://www.gnu.org/software/gettext/manual/html_node/Obsolete-Entries.html)

I am afraind `gopo` does not support obsolete entries.
Is there a good way to import obsolete messages from old po files to new po files?

**Comment by masaori335** [comment]
Yes, I mean "Obsolete Messages".
And I found that the original file (`admin/http-proxy-caching.en.po`) is not updated completely in [Fall 2014 Update PO files](https://github.com/trafficserver-doc-ja/trafficserver-doc-ja/issues/22). So there were already some obsolete messages before your work.

I have no idea to import those messages. But msgid is changed in anyway, so we need fix those translations by our hands if obsolete messages are imported.

**Comment by masaori335** [comment]
Filed this issue in https://github.com/trafficserver-doc-ja/trafficserver-doc-ja/issues/27 Let's discuss at there.
And I'll review this PR.

File: doc/locale/ja/LC_MESSAGES/admin-guide/configuration/cache-basics.en.po [hunk 1]
```diff
@@ -530,6 +544,10 @@ msgid ""
 "information to users on demand. It improves performance and frees up "
 "Internet bandwidth for other tasks."
 msgstr ""
+"HTTPプロキシーキャッシュは頻繁にアクセスされる（ドキュメント、イメージや記事"
```
Comment (masaori335) [inline doc/locale/ja/LC_MESSAGES/admin-guide/configuration/cache-basics.en.po]: Please add a space after `HTTP`? Sorry for bother you, but we have a rule to add a space before and after english words. Details are in https://github.com/trafficserver-doc-ja/trafficserver-doc-ja/issues/12

File: doc/locale/ja/LC_MESSAGES/admin-guide/configuration/cache-basics.en.po [hunk 2]
```diff
@@ -875,6 +912,8 @@ msgid ""
 "Modified`` and ``Date`` headers. To control how long these objects are "
 "considered fresh in the cache, specify an *absolute freshness limit*."
 msgstr ""
+"いくつかのオブジェクトは ``Expires`` ヘッダーを持っていない、あるいは "
+"``Last-Modified`` ヘッダーと ``Date`` ヘッダーの両方を持っていません。"
```
Comment (masaori335) [inline doc/locale/ja/LC_MESSAGES/admin-guide/configuration/cache-basics.en.po]: Please translate a sentence starting with `To control how long...`.

File: doc/locale/ja/LC_MESSAGES/admin-guide/configuration/cache-basics.en.po [hunk 3]
```diff
@@ -925,6 +964,9 @@ msgid ""
 "the age is less than ``max-age``, then the object is fresh and can be "
 "served from the Traffic Server cache."
 msgstr ""
+"サーバーからの ``max-age``  ヘッダーはオブジェクトの年齢と比べられます。年齢"
```
Comment (masaori335) [inline doc/locale/ja/LC_MESSAGES/admin-guide/configuration/cache-basics.en.po]: Please translate "age" as "経過時間". See https://www.transifex.com/jpeach/traffic-server-admin/glossary/ja_JP/

File: doc/locale/ja/LC_MESSAGES/admin-guide/configuration/cache-basics.en.po [hunk 4]
```diff
@@ -597,6 +615,9 @@ msgid ""
 "object is fresh enough to serve. If it is fresh, then Traffic Server serves "
 "it to the client as a *cache hit* (see the figure below)."
 msgstr ""
+"オブジェクトがキャッシュにある場合は、Traffic Serverはオブジェクトが提供する"
+"のに十分新鮮かどうかを確認します。新鮮な場合、Traffic Serverはそれを*キャッ"
```
Comment (masaori335) [inline doc/locale/ja/LC_MESSAGES/admin-guide/configuration/cache-basics.en.po]: Whitespace is required before start-string of inline markup

> 1. The start-string must start a text block or be immediately preceded by whitespace or any of  ' " ( [ { or <.
http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup

File: doc/locale/ja/LC_MESSAGES/admin-guide/configuration/cache-basics.en.po [hunk 5]
```diff
@@ -597,6 +615,9 @@ msgid ""
...
+"シュヒット*としてクライアントに提供します（下図参照）。"
```
Comment (masaori335) [inline doc/locale/ja/LC_MESSAGES/admin-guide/configuration/cache-basics.en.po]: Similar things are required by end-string of inline markup
> The end-string must end a text block (end of document or followed by a blank line) or be immediately followed by whitespace or any of ' " . , : ; ! ? - ) ] } / \ or >.
http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup

**Comment by masaori335** [comment]
@hnakamur Translations are looks good! But there are some style and markup issues, I didn't point out all. Please fix those:)

**Comment by hnakamur** [comment]
@masaori335 Thanks for your review! I fixed issues you pointed out.
Could you take another look?
Thanks!

**Pull request merged**